### PR TITLE
Nk/feature auth

### PR DESCRIPTION
**Pull request by NatalyMac**
To check
Authentication user via AuthService, without logout functionality
to cancel auth logic effect  set ENV_APP = local and you will have authenticated user by default = second user in database.

### CODE REVIEW

File: app/Http/Middleware/AuthMain.php [hunk 1]
```diff
@@ -4,24 +4,33 @@
...
 use Illuminate\Http\Response;
-use Cookie;
-use App\Models\User;
 use Illuminate\Support\Facades\Auth;
-use Emarref\Jwt\Claim;
 
 
 class AuthMain extends AuthService
 {
+    protected $urlAuth = 'http://team.binary-studio.com/auth/';
```
Comment (binaryvo) [inline app/Http/Middleware/AuthMain.php]: Please move the urlAuth to a separated config file. Let's say: /config/authserver.php

Comment (NatalyMac) [reply app/Http/Middleware/AuthMain.php]: done :) check please, 
removed all config data such as 
    "urlAuth" => "http://team.binary-studio.com/auth/",
    "cookieName" => "x-access-token",
    "secretKey" => "superpupersecret",
    "urlUserInfo" => "http://team.binary-studio.com/profile/user/getByCentralId/",

 into the config/authserver.php

File: app/Http/Middleware/AuthService.php [hunk 2]
```diff
@@ -4,31 +4,158 @@
...
+                
+                if ($user->deleted_at != null) {
+                    $user->restore();
+                }
+                $user->first_name = $userInfo[0]['name'];
+                $user->last_name = $userInfo[0]['surname'];
+                $user->global_id = $userInfo[0]['serverUserId'];
+                $user->status_id = 1;
+                $user->save();
+            }
```
Comment (binaryvo) [inline app/Http/Middleware/AuthService.php:138]: Make the retrieving of first element of an array in getUserInfo() method

Comment (NatalyMac) [reply app/Http/Middleware/AuthService.php:138]: done, check pls

**Pull request merged**